frontend/src/pages/summaryPage.tsx

```diff
@@ -16,21 +17,32 @@ export default function SummaryPage({
   const [keywords, setKeywords] = useState<string[]>([]);
   const [generateTime, setGenerateTime] = useState(0);
 
-  const generateSummary = useCallback(() => {
+  const generateSummary = useCallback(async () => {
     const time = Date.now();
     setSummaryIsLoading(true);
+    const summaryId = uuidv4();
     const requestOptions = {
       method: 'POST',
       headers: { 'Content-Type': 'application/json' },
-      body: JSON.stringify({ content: text.join(), length: 3 }),
+      body: JSON.stringify({ content: text.join(), length: 3, summaryId: summaryId }),
     };
-    fetch('http://localhost:3000/summary', requestOptions)
+    const summaryPath = await fetch('https://xi9bq76tv2.execute-api.us-west-2.amazonaws.com/prod/summaryRequest', requestOptions)
       .then((res) => res.json())
       .then((data) => {
-        setSummary([data.summarized_text]);
-        setSummaryIsLoading(false);
-        setGenerateTime(Date.now() - time);
+        return data.summaryPath;
       });
+    const interval = setInterval(() => {
+      fetch(summaryPath).then((res) => {
+        if (res.status === 200) {
```

Comment:
Is it possible to remove the summary.json after the summary state is set? We can reduce a lot of storage usage.

Reply:
It's possible. We can

1. Configure [life cycle policy for s3 object](https://docs.aws.amazon.com/cdk/api/v2/docs/aws-cdk-lib.aws_s3.LifecycleRule.html)
2. Add another API to delete specific S3 objects after we fetched the S3 object.

However, this is out of the scope of this pr since this pr focus on resolving the issue. We have 5GB of free storage, which should be good for now. I can make another issue and work on that after this pr.

Reply:
Yep, sounds good! I'll resolve this comment